scripts/cfde_common.py

```diff
@@ -35,3 +38,17 @@ def write_output_pieces(output_dir, widget_name, cv_id, md, *, verbose=False):
 
     if verbose:
         print(f"Wrote markdown to {output_filename}")
+
+
+def get_portal_page_ids(term):
```

Comment:
Something we could (should?) consider - downloading this once in snakemake for each term, and saving it locally. can you create an issue?

Reply:
see #70

Reply:
made a rule that works well to download files when called directly. will need to add these files as inputs to other rules in order for this to be called with rule all